sdk/search/azure-search-documents/pom.xml

```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.4.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
Would rather not eagerly move to an unreleased version of Azure Core, is this being done to make testing the build from source issue more easily?

Reply:
I am not sure why. The search ci nightly build failed because of new azure core change. 
I don't quite get why the build test on new changes.

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=354178&view=logs&j=3d7a0212-45d0-5080-1981-6d6ef159b344&t=14d7dc25-187c-516a-cf22-fdfcecf8f884

Reply:
Search changes here is to test over the azure-core-test changes. Will revert back to azure-core and azure-core released version after testing.